sample-azure-ha-yamls/azure-ha-kustomize-statefulset.yaml

```diff
@@ -82,8 +82,6 @@ spec:
         env:
         - name: DB_NAME
           value: <db-name>
-       -  name: DB_PORT
-          value: 5432
```

Comment:
Just to confirm, it was removed because the port should be taken from the secret file? 
```
 - name: DB_PORT
          valueFrom:
            secretKeyRef:
              name: db-secret
              key: db-port
```

Reply:
I think the secret file doesn't include this 🤔